federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -191,10 +191,17 @@ async function run() {
         }
       } catch (e) {
         logger.warn(`an error happened in the worker runtime ${e}\n`);
+        let unexpectedError = {
+          name: "unexpectedError",
+          message: `${e}`,
+          extensions: {
+            code: "an error happened in the worker runtime",
```

Comment:
Does this message end up being client facing?  If so, I don't think this will be meaningful enough as it is right now.

Additionally, `code` should be a machine readable code — it's typically something like `GRAPHQL_PARSE_FAILED`.   See the list [of some known codes here](https://www.apollographql.com/docs/apollo-server/data/errors/#error-codes), though I think we have some in Router, too.  Point being, I don't think it should be a message string, which would be on another property (perhaps `message`?)